src/window/error_window.rs

```diff
@@ -0,0 +1,488 @@
+use skia_safe::{
+    canvas::{Canvas, SaveLayerRec},
+    colors::{BLACK, WHITE},
+    textlayout::{
+        FontCollection, Paragraph, ParagraphBuilder, ParagraphStyle, TextHeightBehavior, TextIndex,
+        TextStyle,
+    },
+    Color4f, FontMgr, Paint, Point, Rect, Size,
+};
+use strum::IntoEnumIterator;
+use strum::{EnumCount, EnumIter};
+use winit::{
+    dpi::PhysicalSize,
+    event::{ElementState, Event, KeyEvent, Modifiers, MouseScrollDelta, WindowEvent},
+    event_loop::{EventLoop, EventLoopWindowTarget},
+    keyboard::Key,
+    window::WindowBuilder,
+};
+
+use crate::{
+    clipboard,
+    cmd_line::SRGB_DEFAULT,
+    renderer::{build_context, build_window, GlWindow, WindowedContext},
+    window::{load_icon, SkiaRenderer, UserEvent},
+};
+
+const TEXT_COLOR: Color4f = WHITE;
+const BACKGROUND_COLOR: Color4f = BLACK;
+const FONT_SIZE: f32 = 12.0 * 96.0 / 72.0;
+const PADDING: f32 = 10.0;
+const MAX_LINES: i32 = 9999;
+const MIN_SIZE: PhysicalSize<u32> = PhysicalSize::new(800, 600);
+const DEFAULT_SIZE: PhysicalSize<u32> = PhysicalSize::new(500, 500);
```

Comment:
Mixed up. Their values and their effect are correct, but their names should be exchanged.

```suggestion
const DEFAULT_SIZE: PhysicalSize<u32> = PhysicalSize::new(800, 600);
const MIN_SIZE: PhysicalSize<u32> = PhysicalSize::new(500, 500);
```

Reply:
Fixed, I must have made some mistake when I refactored the variable names.